pkg/asset/k8s.go

```diff
@@ -99,6 +138,7 @@ func newControllerManagerSecretAsset(assets Assets) (Asset, error) {
 	secretAssets := []string{
 		AssetPathServiceAccountPrivKey,
 		AssetPathCACert, //TODO(aaron): do we want this also distributed as secret? or expect available on host?
+		AssetPathCAKey,  // ^
```

Comment:
I'm not really sure what that TODO means -- these are being distributed as secrets...